src/Sylius/Component/Order/spec/Model/OrderSequenceSpec.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Component\Order\Model;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Component\Order\Model\AdjustmentInterface;
+use Sylius\Component\Order\Model\OrderItemInterface;
+use Sylius\Component\Order\Model\OrderItemUnitInterface;
+use Sylius\Component\Order\Model\OrderSequence;
+use Sylius\Component\Order\Model\OrderSequenceInterface;
+
+/**
+ * @mixin OrderSequence
+ *
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+class OrderSequenceSpec extends ObjectBehavior
```

Comment:
We can start introducing a final keyword for all spec classes, according to latest @pamil [changes](https://github.com/Sylius/Sylius/pull/5497/files#diff-dd3791c020f9b7bcab549ef7c7914047R24).